velero-plugins/main.go

```diff
@@ -24,10 +23,6 @@ func newImageStreamBackupPlugin(logger logrus.FieldLogger) (interface{}, error)
 	return &imagestream.BackupPlugin{Log: logger}, nil
 }
 
-func newImageStreamRestorePlugin(logger logrus.FieldLogger) (interface{}, error) {
-	return &imagestream.RestorePlugin{Log: logger}, nil
-}
-
```

Comment:
Same as above -- lets keep this one around since my next PR will be modifying it.